packages/bundler/src/webpack-config.ts

```diff
@@ -163,6 +163,7 @@ export const webpackConfig = ({
 				},
 				{
 					test: /\.jsx?$/,
+					exclude: /node_modules/,
```

Comment:
seriously this fixes the issue?

Reply:
Yes, we try to add fast refresh to all node modules, which is probably not necessary anyway, and the fast refresh fails when doing so, since some node modules are too modern and don't include `module.exports` anymore. At least this is my understanding

Reply:
Got it. Thanks for the awesome explanation. :fire: